business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -4,17 +4,17 @@
 
   <parent>
     <groupId>org.kie</groupId>
-    <artifactId>kie-wb-tests</artifactId>
+    <artifactId>business-central-tests</artifactId>
     <version>7.12.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>kie-wb-tests-gui</artifactId>
+  <artifactId>business-central-tests-gui</artifactId>
 
   <name>KIE (Drools) Workbench Tests :: GUI</name>
-  <description>GUI tests for KIE Workbench and KIE Drools Workbench</description>
+  <description>GUI tests for Business Central and KIE Drools Workbench</description>
 
   <properties>
-    <app.name>kie-wb</app.name><!-- for tests to know if they are running against kie-wb or kie-drools-wb -->
+    <app.name>kie-wb</app.name><!-- for tests to know if they are running against business-central or kie-drools-wb -->
```

Comment:
Please change the property value and update comment like so (to match the change you did in KieSeleniumTest.java)

`<app.name>business-central</app.name><!-- for tests to know if they are running against business-central or business-monitoring -->`

Reply:
Hello,
I tried to make this rename before (also the ENUM), the issue is if I change to kie-wb I would also have to rename kie-wb.jsp to business-central.jsp (the selenium tests).

If you also want to kie-wb.jsp to be renamed we need to decide if it will be business-central.jsp, bc.jsp or something like that

Reply:
Sorry, didn't realize this is about the .jsp name. In that case rather leave it as is.